jest.config.mjs

```diff
@@ -0,0 +1,17 @@
+export default {
+  preset: 'jest-puppeteer',
+
+  // Custom matchers
+  setupFilesAfterEnv: ['./config/jest/matchers.js'],
+
+  // Environment defaults for JSDOM
+  testEnvironmentOptions: {
+    url: 'https://design-system.service.gov.uk'
+  },
+
+  // Enable GitHub Actions reporter UI
+  reporters: ['default', 'github-actions'],
+
+  // Test timeout increased (5s to 15s)
+  testTimeout: 15000
```

Comment:
Is this increase to catch flaky stuff?

Reply:
> Is this increase to catch flaky stuff?

Just noticed we had 15s on `govuk-frontend` so kept it the same!

With us doubling the workers to `--maxWorkers=2` I thought it was sensible